docs/source/workflows/a2a/a2a-client.md

```
@@ -0,0 +1,301 @@
+- Returns string response
+
+**Example:**
+```yaml
+workflow:
+  _type: react_agent
+  tool_names:
+    - currency_agent  # Uses high-level .call() function
+```
+
+The LLM will see a function like:
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)